datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/http.yaml

```diff
@@ -221,6 +221,14 @@
       X-Auth-Token: <AUTH_TOKEN>
     type: object
   description: Additional headers to send with every request.
+- name: host_header_ssl
+  value:
+    example: false
+    type: boolean
+  description: |
+    If set to true, the Host header is used to match 
+    against the SSL Certificate CN or SAN.
+    Requires persist_connections set to true.
```

Comment:
```suggestion
    If a `Host` header is set, this enables its use for SNI (matching against the TLS certificate CN or SAN).
    Requires `persist_connections` set to `true`.
```

Reply:
Updated. I've also removed the following due to implicit persist_connections
_Requires `persist_connections` set to `true`._